plugins/copr.py

```diff
@@ -463,21 +464,49 @@ def _download_repo(self, project_name, repo_filename, chroot=None):
 
         for line in f:
             if re.match(r"\[copr:", line):
-                repo_filename = os.path.join(self.base.conf.get_reposdir,
-                                             "_" + line[1:-2] + ".repo")
+                self.repo_filename = os.path.join(self.base.conf.get_reposdir,
+                                                  "_" + line[1:-2] + ".repo")
             break
 
         # if using default hub, remove possible old repofile
         if self.copr_url == self.default_url:
             # copr:hub:user:project.repo => _copr_user_project.repo
-            old_repo_filename = repo_filename.replace("_copr:", "_copr", 1)\
+            old_repo_filename = self.repo_filename.replace("_copr:", "_copr", 1)\
                 .replace(self.copr_hostname, "").replace(":", "_", 1).replace(":", "-")\
                 .replace("group_", "@")
             if os.path.exists(old_repo_filename):
                 os.remove(old_repo_filename)
 
-        shutil.copy2(f.name, repo_filename)
-        os.chmod(repo_filename, stat.S_IRUSR | stat.S_IWUSR | stat.S_IRGRP | stat.S_IROTH)
+        shutil.copy2(f.name, self.repo_filename)
+        os.chmod(self.repo_filename, stat.S_IRUSR | stat.S_IWUSR | stat.S_IRGRP | stat.S_IROTH)
+
+    def _runtime_deps_warning(self, project_name, chroot=None):
+        repo_file = ConfigParser()
+        repo_file.read(self.repo_filename)
+
+        if len(repo_file.sections()) == 1:
+            return
+
+        runtime_deps = []
+        for section in repo_file.sections():
+            if section in self.repo_filename:
+                continue
+            runtime_deps.append(section)
+
+        print(_("The Copr project {0} uses the following repositories as runtime dependencies:"
+              .format(project_name)))
+        for dep in runtime_deps:
+            print(dep)
+
+        msg = _("Do you want to keep them enabled?")
+        if self.base._promptWanted():
+            if self.base.conf.assumeno or not self.base.output.userconfirm(
+                    msg='{} [y/N]: '.format(msg), defaultyes_msg='{} [Y/n]: '.format(msg)):
```

Comment:
Does this mean that if `dnf -y` isn't used, default is no?  (that would be correct)

Reply:
That's correct.